app/src/main/res/layout/option_story_text_size.xml

```diff
@@ -13,12 +16,12 @@
     android:id="@+id/reading_text_size_item_layout"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:background="@drawable/general_item_background_border"
+    android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"
```

Comment:
@shankarpriyank actually problem here is that in tablet layout by default 1st element is selected, but that's not the same for mobile thus it shows unexpected results. I think we can just merge mobile and tablet layouts separately and not together. @rt4914 I am not able to figure out any possible solution for this case except above, can you pls suggest any.